pkg/controller/secret-mirror_test.go

```diff
@@ -6,6 +6,8 @@ import (
 	"testing"
 	"time"
 
+	conf "github.com/openshift/ci-secret-mirroring-controller/pkg/controller/config"
```

Comment:
Just use `config`? Why alias this?

Reply:
Done.
Intellij complaint:

https://github.com/openshift/ci-secret-mirroring-controller/blob/00e5b22eeb427dc8dee2c73f19ec896270cba453/pkg/controller/secret-mirror_test.go#L36

need to rename either the pkg or the var here:

https://github.com/openshift/ci-secret-mirroring-controller/blob/00e5b22eeb427dc8dee2c73f19ec896270cba453/pkg/controller/secret-mirror_test.go#L22